test/api/v3/integration/groups/PUT-groups.test.js

```diff
@@ -3,6 +3,7 @@ import {
   generateUser,
   translate as t,
 } from '../../../../helpers/api-v3-integration.helper';
+import {MAX_SUMMARY_SIZE_FOR_GUILDS} from '../../../../../website/common/script/constants.js';
```

Comment:
shouldn't this check be at the model level instead of doing it for every route?

Reply:
See: https://github.com/HabitRPG/habitica/issues/9244#issuecomment-351550431

> So, I think the best start here is to validate length during the api request.

Reply:
I dont know what the reason for this is. Though if requested I could move it to the model level instead. Seems more reasonable…

Reply:
Hmm I see, I think the problem might be that we don't want to start throwing errors if someone had a longer summery before the validation is introduced